src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,39 @@
+import { useParams, Link } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { getArticleById } from '../utils/api';
+
+export const SingleArticle = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id).then((articleData) => {
+      setArticle(articleData);
+    });
+  }, []);
+
+  return (
+    <>
+      <div className='single-article'>
+        <Link to={`/`}>
+          <h3 className='article-header'>{article.title}</h3>
+        </Link>
```

Comment:
From a user point of view, I'm not sure if this should be a link. If we're navigating directly to view a specific article then we may just want this to be the `<h3>` itself. Clicking on an article title to navigate to the home screen to view all titles doesn't seem intuitive :)

Reply:
This is no longer a link